Shadows.md

```diff
@@ -0,0 +1,62 @@
+| Apple: Native macOS </br></br> [shadowBlurRadius](https://developer.apple.com/documentation/uikit/nsshadow/1429846-shadowblurradius) documentation | <img width="443" alt="NativeMacOS_100" src="https://user-images.githubusercontent.com/78454019/189423358-4223d549-e589-4aa3-abc7-7b1e9a6ef04e.png"> <img width="433" alt="NativeMacOS_50" src="https://user-images.githubusercontent.com/78454019/189423369-ad2926d0-a4b9-448b-afca-38dd6b00a0cb.png"> | <img width="472" alt="NativeMacOS_100_screenshot" src="https://user-images.githubusercontent.com/78454019/189423396-394970d4-6507-419f-bd97-d26929cb6139.png"> <img width="460" alt="NativeMacOS_50_screenshot" src="https://user-images.githubusercontent.com/78454019/189423409-dcb62f08-d139-4193-9f55-30c3f9e2b47d.png"> |
+| Android: React Native Android </br></br> Can test shadows with this [snack](https://snack.expo.dev/@lyzhan/shadow-ios-web-android). Uses elevation, which is just different from both how web/iOS handle shadows| <img width="311" alt="Screen Shot 2022-09-07 at 1 42 18 PM" src="https://user-images.githubusercontent.com/78454019/188983536-e4a17f29-456f-411d-968d-43e32a93f1b6.png"> | <img width="200" alt="Screen Shot 2022-09-06 at 1 46 26 PM" src="https://user-images.githubusercontent.com/78454019/188753672-c8d3853f-7ea6-4b00-b2f6-b79884f30356.png"> |
+
+## Examples
+
+In Fluent UI React Native, here is a snippet of the code that takes a shadowToken object (basically an object that describes shadows the way web does, with blur) and converts it into a react native shadow.
+
+```typescript
+const appleShadowBlurAdjustment = 0.5;
+const defaultShadowBlurAdjustment = 1;
+
+function getShadowTokenStyleSet(shadowToken: ShadowToken) {
+  const keyShadow = shadowToken.key;
+  const ambientShadow = shadowToken.ambient;
+  const shadowBlurAdjustment = Platform.OS === 'macos' || Platform.OS === 'ios' ? appleShadowBlurAdjustment : defaultShadowBlurAdjustment;
+
+  return {
+    key: {
+      shadowColor: shadowColorFromRGBAColor(keyShadow.color),
+      shadowOpacity: shadowOpacityFromRGBAColor(keyShadow.color),
+      shadowRadius: keyShadow.blur * shadowBlurAdjustment,
+      shadowOffset: {
+        width: keyShadow.x,
+        height: keyShadow.y,
+      },
+    },
+    ambient: {
+      shadowColor: shadowColorFromRGBAColor(ambientShadow.color),
+      shadowOpacity: shadowOpacityFromRGBAColor(ambientShadow.color),
+      shadowRadius: ambientShadow.blur * shadowBlurAdjustment,
+      shadowOffset: {
+        width: ambientShadow.x,
+        height: ambientShadow.y,
+      },
+    },
+  };
+}
+```
+
+Additional work:
+- todo: add FluentUI Apple helper function here
+- todo: add Win32 code + screenshot as well
```

Comment:
I think we don't really need these sections or TODOs anymore. We _can_ link the actual files in FURN and FUA, but I think the screenshots move are sufficient. Excellent work!

Reply:
Yeah I'm not really tied to adding the code, I thought I could copy over the method in FURN without it being so long but it ended up just being most of the method. I'm actually thinking I'll just get rid of it altogether, just remembered I think we're maybe going to change the token values instead of adding a helper in FURN/FUA? And the other todo I had was win32 which I will add, so I can get rid of that section altogether